src/main/java/org/example/Game.java

```diff
@@ -1,181 +1,262 @@
   }
 
-    private  void startPlay()  {
+  private void postStartup() {
+    setPlayerSymbol();
+    cyborg.setSymbol((player.getSymbol() == X_SYMBOL) ? O_SYMBOL : X_SYMBOL);
+    boardobj.initializeBoard();
+    boardobj.showBoard();
+    startPlay();
+  }
 
-        if (player.getSymbol() == 'X') {
-            while (true) {
-                playerPlay();
-                computerPlay();
-            }
+  private void startPlay() {
 
-        } else {
-            while (true) {
-                computerPlay();
-                playerPlay();
-            }
-        }
+    if (player.getSymbol() == X_SYMBOL) {
+      while (true) {
+        playerPlay();
+        computerPlay();
+      }
 
+    } else {
+      while (true) {
+        computerPlay();
+        playerPlay();
+      }
     }
+  }
 
+  private int getPositionFromUser() {
+    int position = -1;
 
+    do {
+      try {
 
-    private  void computerPlay() {
+        String result =
+            String.format(
+                "%s enter empty location for the next move from 1 to 9 : ", player.getName());
+        System.out.println(result);
+        position = this.sc.nextInt();
+        if (!(position >= 1 && position <= 9)) {
+          position = -1;
+          throw new OutOfRange(" \uD83D\uDE4F Plzz enter position between 0 to 9 only !");
+        } else if (!boardobj.getEmptyPositions().contains(position)) {
+          position = -1;
+          throw new OutOfPosition(" \uD83D\uDE4F Plzz enter an empty position !");
+        }
 
-        Iterator<Integer> i = boardobj.getIntList().iterator();
-        int position = boardobj.getIntList().get(0);
-        boardobj.getIntList().remove(0);
-        int row = (position - 1) / 3;
-        int col = position - (row * 3) - 1;
-        boardobj.getBoard()[row][col] = cyborg.getSymbol();
-        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~  Cyborg's Move  ~~~~~~~~~~~~~~~~~~~~~~~~~~");
-        boardobj.showBoard();
-        isWinningMoveOrTie(cyborg.getSymbol());
-    }
+      } catch (InputMismatchException e) {
+        println(" \uD83D\uDE4F Plzz Enter only numbers  from 1 to 9 !! ");
+
+      } catch (OutOfRange | OutOfPosition e) {
+        println(e.getMessage());
+
+      } finally {
+        sc.nextLine();
+      }
+
+    } while (position == -1);
+    return position;
+  }
+
+  private int getComputerMove() {
+    return boardobj.getEmptyPositions().get(0);
+  }
 
+  private void computerPlay() {
+    int position = getComputerMove();
+    removePositionFromPositionList();
+    updateBoard(position, cyborg.getSymbol());
+    printUserMove("Cyborg");
+    if (boardobj.getEmptyPositions().size() <= 4) {
+      isWinningMoveOrTie(player.getSymbol());
+    }
+  }
 
-    private void playerPlay()  {
-        int position = -1;
-        Scanner sc = new Scanner(System.in);
+  private List<Integer> convertPositionIntoRowCol(int position) {
+    int row = (position - 1) / 3;
+    int col = position - (row * 3) - 1;
```

Comment:
Try not to use integer constants directly.

Reply:
Here N will come wherever no. 3 is written